File: frontend/src/app/actors/actor.ts
export interface actorDTO{
    id: number
    name: string
    birthDate: Date
    photo: string
    biography: string
}

export interface createActorDTO{
    name: string
    birthDate: Date
    photo: File
    biography: string
}

export interface actorMovieDTO{
    id: number;
    name: string;
    character: string;
    photo: string;
}

File: backend/backend/Entities/Actor.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Entities
{
    public class Actor
    {
        public int Id { get; set; }
        [Required]
        [StringLength(maximumLength: 200)]
        public string Name { get; set; }
        public string Biography { get; set; }
        public DateTime BirthDate { get; set; }
        public string Photo { get; set; }
    }
}
